### DISCUSSION TITLE
Qwik Version

### DISCUSSION THREAD

**Discussion by jdgamble555** [AgnosticUI/agnosticui#249] [Ideas]
It would be cool to make a version for Qwik JS. There doesn't seem to be many UI packages for it.

J

**Comment by roblevintennis** [comment]
I agree but at this point I'm probably going to need to start getting offers from the community to realistically add more packages. I'm already struggling to get time to fix bugs and add requisite features here. Will keep this around though -- definitely Qwik is a cool framework from an obviously talented team ;-)

**Comment by acoreyj** [comment]
I could help but will be a slow process probably. The components are created manually right? Just the css is added as an import via a script?

**Comment by roblevintennis** [comment]
I'm closing out legacy discussions as the v2 rewrite is almost ready and is a complete rewrite overhaul so discussions are no longer relevant at this point.
